doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -36,16 +36,17 @@ create_role ::=
     'CREATE' 'ROLE' role_name ('LOGIN' | 'NOLOGIN' | 'SUPERUSER' | 'NOSUPERUSER')*
 create_schema ::=
     'CREATE' 'SCHEMA' ('IF NOT EXISTS')? schema_name
-    create_sink ::=
-       'CREATE SINK' 'IF NOT EXISTS'? sink_name
-       'FROM' item_name
-       'INTO' (
-        sink_kafka_connector |
-       'AVRO OCF' path-prefix
-       )
-       ('ENVELOPE' ('DEBEZIUM'|'UPSERT'))?
-       ('WITH SNAPSHOT' | 'WITHOUT SNAPSHOT')?
-       ('AS OF' timestamp_expression)?
+create_sink ::=
```

Comment:
While you're updating the BNF, would you mind adding a description of what these fields do to the sinks docs, as well?